stake-pool/js/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import * as schema from './schema.js';
+import borsh from 'borsh';
+import solanaWeb3 from '@solana/web3.js';
+
+export class StakePool {
+  /**
+   * Wrapper class for a stake pool.
+   * Each stake pool has a stake pool account and a validator list account.
+   * (Optionally) a stake pool can also have a name and ticker.
+   */
+  name: string;
+  ticker: string;
```

Comment:
These aren't stake pools per se, these are "combined" stake pools that have both stake pool and validator list accounts. Dom wants each stake pool to have a human-readable name and ticker. I can remove them if you'd like?

Reply:
Hm, seems like this part should be provided by another service.  Where are the human-readable name / ticker supposed to come from?  If we have a name anywhere, I thought it would be on the mint for the stake pool token using one of the token metadatas.

Reply:
I think the idea is for stake pool managers to give us their desired stake pool name and ticker. Using the token metadata is a good idea.